src/geo/transform.js

```diff
@@ -574,11 +575,11 @@ class Transform {
     }
 
     /**
-     * Return a zoom level that will cover all tiles the transform
-     * @param {Object} options options
+     * Return a zoom level that will cover all tiles the transform.
```

Comment:
This sentence looks incomplete, and I'm not sure what it's trying to convey. @mapbox/gl-js team, any thoughts?

Reply:
Maybe
```suggestion
     * Return the highest zoom level that fully includes all tiles in the transform.
```
I'm not 100% sure that's accurate so hopefully someone else will chime in!

Reply:
Is it instead "the *smallest* integer zoom level which covers all tiles within the transform's boundaries." (Sorta like the [supremum (least upper bound)](https://en.wikipedia.org/wiki/Infimum_and_supremum) if you're into math.) I think "cover" is an valid word here since it's heavily used within the source, and since (correct me if I'm wrong) this jsdoc material does not make it into the public docs.

Reply:
This seems like it's on the right track! 

> the _smallest_ integer zoom level which covers all tiles within the transform's boundaries.

Would it be the _largest_ though, since a zoom of 0 would cover all tiles for any transform?

Reply:
Gah, you're absolutely right! I'm mixing myself up… You're right. @SnailBones' initial wording was correct. Highest = if you went higher, you'd zoom in too far and not cover it. 👍